src/pages/tours/create.tsx

```diff
@@ -0,0 +1,20 @@
+import AppPageLayout from "../../layouts/app-page-layout";
+import Typography from "@mui/material/Typography";
+import TourForm from "../../components/app/TourForm";
+import {Tour} from "../../types/tour";
+import {Point} from 'geojson'
+
+const NewTour = () => {
+    const tour: Tour = new Tour("", "", {} as Point, {} as Point, "", new Date(), new Date())
+
```

Comment:
This page is essentially public since it is not using `withAuthenticatedOrRedirect`.